pkg/preflight/assets/host-preflights.yaml

```diff
@@ -20,17 +20,23 @@ spec:
     - cpu:
         checkName: "Number of CPUs"
         outcomes:
-          - fail:
+          - warn:
               when: "count < 4"
-              message: At least 4 CPU cores are required
+              message: At least 4 CPU cores are recommended
+          - fail:
+              when: "count < 2"
+              message: At least 2 CPU cores are required
```

Comment:
```suggestion
              message: At least 2 CPU cores are required, and 4 CPU cores are recommended
```

Reply:
It's a poor experience to be told a minimum, then reconfigure the machine to meet the minimum, only to be then told a higher recommended number, so just give them both at the same time.

My only question is, will it be clear to the user that they're below both? Does it say how many cores we detect?